src/main/webapp/resources/js/pages/launch/LaunchButton.jsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import {
+  IconLaunchPipeline,
+  IconPlusCircle,
+} from "../../components/icons/Icons";
+import { SPACE_LG } from "../../styles/spacing";
+import { Button } from "antd";
+import { useLaunch } from "./launch-context";
+
+export function LaunchButton({ disabled, loading }) {
+  const [{ automatedId }] = useLaunch();
+  return automatedId ? (
+    <Button
```

Comment:
Instead of duplicating basically the same html for both buttons I think it may be better to just have html for one button and before the return set the button text and icon depending on if there is an automated id or not. Thoughts?

Reply:
LOL, I did that in 1e99633 before I read this.  Great minds think alike

Reply:
Haha that they do!